src/app/external-cluster-wizard/steps/external-cluster/provider/aks/template.html

```diff
@@ -14,137 +14,164 @@
+        Node Pool Name must contain only lowercase letters and numbers, must be 12 or fewer characters.</mat-error>
     </mat-form-field>
-  </div>
-
-  <div fxFlex="50%"
-       fxLayout="column"
-       fxLayoutGap="8px">
-    <mat-card-header class="km-no-padding">
-      <mat-card-title>Node Pool Setting</mat-card-title>
-    </mat-card-header>
 
     <km-autocomplete label="VM Size"
                      required="true"
                      [isLoading]="isLoadingVmSizes"
                      [options]="vmSizes"
                      [formControlName]="Controls.VmSize">
-      <ng-container hint>
+      <ng-container *ngIf="!isDialogView()"
+                    hint>
         VM size availability varies by region.
         <a href="https://docs.microsoft.com/azure/aks/quotas-skus-regions"
            rel="noopener noreferrer"
            target="_blank">supported versions</a>.
       </ng-container>
+      <ng-container *ngIf="isDialogView()"
+                    hint>
+        set the VM Size for this Node.
+      </ng-container>
     </km-autocomplete>
 
-    <mat-form-field>
-      <mat-label>Node Pool Name</mat-label>
-      <input matInput
-             type="text"
-             autocomplete="off"
-             [formControlName]="Controls.NodePoolName" />
-      <mat-hint>Enter node pool name. It must contain only lowercase letters and numbers.</mat-hint>
-      <mat-error *ngIf="form.get(Controls.NodePoolName).hasError(ErrorType.Required)">
-        Node Pool Name is <strong>required</strong>.
-      </mat-error>
-    </mat-form-field>
+    <km-autocomplete *ngIf="isDialogView()"
+                     label="Kubernetes Version"
+                     required="true"
+                     [options]="nodePoolVersions"
+                     [formControlName]="Controls.KubernetesVersion">
+      <ng-container hint>
+        Set the kubernetes version.
+      </ng-container>
+    </km-autocomplete>
 
     <km-number-stepper [formControlName]="Controls.Count"
                        label="Count"
                        mode="all"
                        required="true"
                        [hint]="form.get(Controls.EnableAutoScaling).value ?
-                        'Number of agents (VMs) to host docker containers can be between min and max auto scaling config' :
-                        'Number of agents (VMs) to host docker containers' "
+                        'Number of Nodes must be in the range of min and max auto scaling config' :
+                        'Number of Nodes' "
                        [min]="form.get(Controls.MinCount).value"
                        [max]="form.get(Controls.MaxCount).value">
     </km-number-stepper>
 
-    <mat-form-field>
-      <mat-label>Mode</mat-label>
-      <input matInput
-             type="text"
-             autocomplete="off"
-             [formControlName]="Controls.Mode" />
-    </mat-form-field>
+    <label class="km-radio-group-label">Select Mode</label>
```

Comment:
Not readable styles. It will be great to have color with less opacity?

https://user-images.githubusercontent.com/17727069/180957354-45afe65a-0c1a-45ff-800c-a1482349c9ac.mp4